melissa/actions/repeat.py

```diff
@@ -0,0 +1,11 @@
+# Melissa
+from melissa import profile
+from melissa.tts import tts
+
+
+WORDS = {'repeat_text': {'groups': ['repeat']}
```

Comment:
Awesome, works well @anurag-ks. We accept a single commit per PR. Can you rebase both these commits and squash them to a single one?